src/main.rs

```diff
@@ -260,6 +261,12 @@ async fn handle_request(
     let resp = match client.request::<Vec<u8>>(raw_request).await {
         Ok(resp) => resp,
         Err(e) => {
+            #[cfg(feature = "expose-metrics")]
+            if let TwilightErrorType::Response { status, .. } = e.kind() {
+                let end = Instant::now();
+
+                histogram!(METRIC_KEY.as_str(), end - start, "method"=>m.to_string(), "route"=>p, "status"=>status.to_string());
```

Comment:
```suggestion
                histogram!(
                    METRIC_KEY.as_str(),
                    end - start,
                    "method" => m.to_string(),
                    "route" => p,
                    "status" => status.to_string(),
                );
```
How do you feel about this? I feel the readability of this line is important.

Reply:
Should we do te same at te iter instance of tat line?